.circleci/config.yml

```diff
@@ -36,13 +36,9 @@ jobs:
 
   test_migrations_from_6_0:
       machine:
-          image: ubuntu-2004:202101-01
+          image: ubuntu-2204:2022.07.1
       steps:
           - checkout
-          - run:
```

Comment:
We don't need that anymore?

Reply:
Nope because I bumped the docker image and in this new image ubuntu and circleci user does not have the same uid (ubuntu => 1000 and circleci => 1001) before circleci (the user that launch the command) uuid was 1000. 

As `circleci/test_migrations_from_6_0.sh` made a checkout the project owner should be circleci, the owner is given to user 1000 in the `circleci/test_migrations_from_6_0.sh` just after made a git checkout